src/psyclone/transformations.py

```diff
@@ -960,29 +1002,32 @@ def apply(self, node, reprod=None):
         :py:meth:`OMPLoopDirective.gen_code` is called), this node must be
         within (i.e. a child of) an OpenMP PARALLEL region.
 
-        The optional reprod argument will cause a reproducible
-        reduction to be generated if it is set to True, otherwise the default
-        value (as read from the psyclone.cfg file) will be used. Note,
-        reproducible in this case means obtaining the same results
+        If the keyword "reprod" is specified in the options, it will cause a
+        reproducible reduction to be generated if it is set to True, otherwise
+        the default value (as read from the psyclone.cfg file) will be used.
+        Note, reproducible in this case means obtaining the same results
         with the same number of OpenMP threads, not for different
         numbers of OpenMP threads.
 
         :param node: the supplied node to which we will apply the \
-        OMPLoopTrans transformation
+                     OMPLoopTrans transformation
         :type node: :py:class:`psyclone.psyGen.Node`
-        :param reprod: optional argument to determine whether to \
-        generate reproducible OpenMP reductions (True) or not \
-        (False). The default value is None which will cause PSyclone \
-        to look up a default value
-        :type reprod: Boolean or None
+        :param options: a dictionary with options for transformations\
+                        and validation. This transformation accepts "reprod",\
```

Comment:
Please document this "reprod" option using the same form as elsewhere, e.g.:

:param bool options["reprod"]: blah blah

Reply:
Done.